.circleci/config.yml

```diff
@@ -274,7 +274,7 @@ jobs:
       - deploy:
           name: Deploy to live test
           command: |
-            sed -i -e s/:latest/:$CIRCLE_SHA1/ deploy/test/cron-*.yaml
+            # sed -i -e s/:latest/:$CIRCLE_SHA1/ deploy/test/cron-*.yaml
```

Comment:
Did you mean to comment this out but leave it in?

Reply:
Good catch, thanks. Removing it